guide/how-it-works/payment-request-formats.md

```diff
@@ -9,129 +9,253 @@ redirect_from:
  - /guide/payments/send/payment-request-formats/
  - /guide/payments/request/payment-request-formats/
 main_classes: -no-top-padding
-image: /assets/images/guide/payments/send/bitcoin-payments-social.jpg
-image_base: /assets/images/guide/payments/send/
-images:
-    - file: home
-      alt:
-      caption:
+image: /assets/images/guide/how-it-works/payment-request-formats/payment-request-formats-preview.jpg
+
 ---
 
 <!--
 
-Editor's notes
+Illustration source: https://www.figma.com/file/qr4P17z6WSPADm6oW0cKw2/?node-id=937%3A6451
 
 -->
 
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/payment-request-formats/payment-request-formats.jpg"
+   retina = "/assets/images/guide/how-it-works/payment-request-formats/payment-request-formats@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/payment-request-formats/payment-request-formats-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/payment-request-formats/payment-request-formats-mobile@2x.jpg"
+   alt-text = "Header image of the payment request formats page"
+   width = 1600
+   height = 500
+   layout = "full-width"
+%}
+
 # Payment request formats
 
-Payment information can be shared in many formats and over diverse communication channels. Each has its advantages and limitations, these formats seen together allow for broad flexibility in initiating payments. Some of them are still maturing and have varying support across applications.
+Payment information can be shared in many formats and over diverse communication channels. Each has its advantages and limitations. These formats seen together allow for broad flexibility in initiating payments, both on lightning and on-chain. Some of them are still maturing and have varying support across applications.
+
+{% include tip/open.html color="blue" icon="info" label="Test your apps for compatibility" %}
+
+If you need to test whether your application supports a payment request type, you can try scanning the examples shown on this page. If your application does not support a particular format, it should show the user a friendly error message.
+
+{% include tip/close.html %}  
+
+## Uniform resource identifier (URIs) schemes
+
+A [Uniform Resource Identifier](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) (URI) is a web technology that uses unique prefixes to identify anything. This can be real-world objects, websites, information, etc.
+
+Bitcoin uses URIs to help apps identify payment requests. The technical details of this scheme are detailed in [BIP 21](https://bips.xyz/21).
+
+BIP 21 uses a `bitcoin:` URI scheme to identify and help applications manage bitcoin payment requests. Applications should support this if they want to be [interoperable]({{'/guide/getting-started/principles/#interoperability' | relative_url}}) with other bitcoin products. Some other less common, non-standardized, URI schemes used for bitcoin payment requests include:
```

Comment:
```suggestion
BIP 21 uses a `bitcoin:` URI scheme to identify and help applications manage bitcoin payment requests. Applications should support this if they want to be [interoperable]({{'/guide/getting-started/principles/#interoperability' | relative_url}}) with other bitcoin products. Some other URI schemes used for bitcoin payment requests include `lightning` and LNURL schemes such as `lnurlp`. Note that a URI scheme can be presented as lowercase or uppercase.
```

I don't agree with the statement the way it's worded.

- [BIP21](https://en.bitcoin.it/wiki/BIP_0021) specifically says it's OK to uppercase the scheme, so that's "standardized".
- All URI schemes are allowed to be uppercase or lowercase according to [RFC 3986](https://www.rfc-editor.org/rfc/rfc3986), so really uppercasing any URI scheme should be considered standardized.
- `lightning:` is defined in [BOLT11](https://en.bitcoin.it/wiki/BIP_0021), so also standardized.
- Not sure I am aware of `lnurl:`, but things like `lnurlc:`, `lnurlp:`, `lnurlw:`, and `keyauth:` are defined in LUD-17, so those would also be standardized.

Reply:
Also note that my very granular focus on these details may not affect the life of a designer very much. 🙂 Just making sure technical claims are accurate.